### ISSUE TITLE
Syntax conflict

### ISSUE THREAD

**Issue by minasvisual** [rgamba/tonic#11]
Hi,
I have a issue with a syntax tag. Always i try call the variable $supervisor.user_name the code understand the "or" as compare flag. This break the code.

{$supervisor.user_name}
Change to:
 `if(@$supervis||.user_name): 
To
`try{ echo @self::callModifier("contextOutTag",$supervisor['user_name'],""); } catch(\Exception $e) { echo $e->getMessage(); }

**Comment by adaniello** [comment]
See my suggestion at https://github.com/rgamba/tonic/issues/10#issuecomment-464032515

**Comment by minasvisual** [comment]
> See my suggestion at [#10 (comment)](https://github.com/rgamba/tonic/issues/10#issuecomment-464032515)

Yes you`re right! I just changed the var name to solve, but this issue need to be commit it  to the repository.

Thanks!
